tests/ops/service/privacy_request/test_request_runner_service.py

```diff
@@ -2133,14 +2142,25 @@ def test_create_and_process_access_request_bigquery(
     assert results[address_table_key][0]["city"] == city
     assert results[address_table_key][0]["state"] == state
 
+    employee_table_key = f"bigquery_example_test_dataset:employee"
+    assert len(results[employee_table_key]) == 1
+    assert results["bigquery_example_test_dataset:employee"] != []
+    assert (
+        results[employee_table_key][0]["address_id"] == bigquery_resources["address_id"]
+    )
+    assert (
+        results[employee_table_key][0]["email"] == bigquery_resources["employee_email"]
+    )
+    assert results[employee_table_key][0]["id"] == bigquery_resources["employee_id"]
+
     pr.delete(db=db)
 
 
 @pytest.mark.integration_external
 @pytest.mark.integration_bigquery
 @pytest.mark.parametrize(
     "dsr_version",
-    ["use_dsr_3_0", "use_dsr_2_0"],
+    ["use_dsr_2_0", "use_dsr_3_0"],
```

Comment:
I didn't do this for any particular reason, I as just testing with one while I was developing the test, and then pasted the other back, but in a different order clearly